ikarus/finiteelements/feresulttypes.hh

```diff
@@ -252,4 +252,19 @@ auto toString() {
 template <template <typename, int, int> class RT1, template <typename, int, int> class RT2>
 constexpr bool isSameResultType = std::is_same_v<Impl::DummyRT<RT1>, Impl::DummyRT<RT2>>;
 
+/**
+ * \brief Meta function to test whether a given ResultType is in a tuple of supported ResultTypes
+ * \tparam SupportedRT tuple type of supported RestultTypes (exported by the elements)
+ * \tparam RT ResultType template to be checked
+ */
+template <typename SupportedRT, template <typename, int, int> class RT>
+static consteval bool isSupportedResultType() {
+  bool result{};
+  Dune::Hybrid::forEach(SupportedRT(), [&]<typename RT_>(RT_ rt) {
+    if constexpr (isSameResultType<RT, RT_::template Rebind>)
```

Comment:
i think using https://en.cppreference.com/w/cpp/types/disjunction is a better fit here. See also https://godbolt.org/z/Yz9hh79eG .

Reply:
thats nice, thanks

Reply:
I am not sure how this implementation relates to the already present implementation here 
https://github.com/ikarus-project/ikarus/blob/0faecc7bf3b415f83861c45b3da704ed29b7d801/ikarus/utils/traits.hh#L108

Both seem to work

Reply:
Okay nvm apperarenty the implementation in traits can not be evaluated to false without a compiler error

Reply:
What do you mean?